MapboxNavigation/RouteMapViewController.swift

```diff
@@ -137,6 +137,10 @@ class RouteMapViewController: UIViewController {
         mapView.camera = tiltedCamera
         mapView.setUserTrackingMode(.followWithCourse, animated: true)
         mapView.logoView.isHidden = false
+        
+        guard let controller = routePageViewController.currentManeuverPage else { return }
+        controller.step = currentStep
+        routePageViewController.notifyDidReRoute()
```

Comment:
`notifyDidReRoute()` is quite a misnomer. I’m afraid that we may in the future introduce logic to `notifyDidReRoute()` that only makes sense for rerouting and now recentering. Maybe we should rename the method now to avoid future surprises.